app/src/main/java/com/google/android/stardroid/activities/util/AbstractGooglePlayServicesChecker.java

```diff
@@ -93,4 +94,28 @@ public void done() {
     Log.d(TAG, "Location rationale Dialog has been shown");
     requestLocationPermission();
   }
+  
+  private void showAlertOnLocationDenied() {
+    AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(parent);
+    alertDialogBuilder.setTitle("Warning!");
+    alertDialogBuilder.setMessage("Are you sure you want to disable auto location? You may then need to set location manually.");
```

Comment:
Sorry - I can't accept changes with hard coded strings - they need to be localized.